### ISSUE TITLE
Support for other unstructured data like pdf, word etc.,

### ISSUE THREAD

**Issue by gnthaker** [instructlab/community#309]
It would be good if we can provide PDF or other unstructured data from which we can generate synthetic data.

**Comment by lhawthorn** [comment]
Thank you for filing this issue, @gnthaker! Helps us keep track of it. We discussed in last night's triage meeting that we desire to also create better tooling for PDF --> Markdown conversion and generally make data ingestion a less cumberson process. As we are moving fast and a young project, I am not sure where on our roadmap this will land timing-wise.

Once again, thank you for filing this issue and assuring we don't lose track of this clear need.

**Comment by lhawthorn** [comment]
The people in the community who I know who have talked the most about this need are on the Triage team. If you want to talk to them about scoping this work, you can find them in #triage on [InstructLab Slack](https://github.com/instructlab/community/blob/main/InstructLab_SLACK_GUIDE.md).

**Comment by jjasghar** [comment]
Yep, @gnthaker, please reach out; we have some thoughts and suggestions to get something off the ground, but nothing formalized in a pipeline or anything.
